setup.py

```diff
@@ -28,9 +28,39 @@
 from distutils.extension import Extension
 from distutils.command.install_data import install_data
 from distutils.command.install_headers import install_headers
+import distutils.ccompiler
 from Cython.Distutils import build_ext
 
 
+
+# Distutils optimizations
+# -----------------------
+
+def parallelCCompile(self, sources, output_dir=None, macros=None,
```

Comment:
Minor: the parallel build is not making a large/significant difference because the different extensions are still done in serial. The main bottleneck for each extension is building cext.so, while compiling the other cpp files takes very little time.